chia/wallet/did_wallet/did_wallet.py

```diff
@@ -189,18 +193,18 @@ def type(cls) -> uint8:
     def id(self):
         return self.wallet_info.id
 
-    async def get_confirmed_balance(self, record_list=None) -> uint64:
+    async def get_confirmed_balance(self, record_list=None) -> uint128:
```

Comment:
Rather than guessing what size integer should be used for which balance/weight/iters/whatever, we should really have some type definitions to make this easier to reason about. For instance, a WalletBalance type, or maybe just MojoAmount or something. Not suggesting a change here, just dropping a thought.

Reply:
Yes! We should name this important type.